panther_power_control/src/relays_node.py

```diff
@@ -1,39 +1,52 @@
 #!/usr/bin/python3
 
+from dataclasses import dataclass
 import RPi.GPIO as GPIO
-from time import sleep, time
 
 import rospy
 
 from geometry_msgs.msg import Twist
 from std_msgs.msg import Bool
 from std_srvs.srv import Trigger, TriggerRequest, TriggerResponse
 
-STAGE2_INPUT = 22
-MOTOR_ON = 6
 
+@dataclass
+class PatherGPIO:              
+    MOTOR_ON = 6       # Pin to enable motor controllers 
+    STAGE2_INPUT = 22  # Check if power can be forwarded to motor controllers
+
+    def __setattr__(self, name: str, value: int) -> None:
+        raise AttributeError(f'Can\'t reassign constant {name}')
 
-class RelaysNode:
-    def __init__(self, name) -> None:
-        self._setup_gpio()
 
+class RelaysNode:
+    def __init__(self, name: str) -> None:
         rospy.init_node(name, anonymous=False)
 
+        self._pins = PatherGPIO()
+        self._setup_gpio()
+        
         self._e_stop_state = False
-        self._cmd_vel_msg_last_time = time()
+        self._cmd_vel_msg_time = rospy.get_time()
 
         # -------------------------------
         #   Publishers
         # -------------------------------
 
-        self._motor_on_pub = rospy.Publisher('hardware/motor_on', Bool, queue_size=1)
-        self._e_stop_state_pub = rospy.Publisher('hardware/e_stop', Bool, queue_size=1)
+        self._motor_on_pub = rospy.Publisher(
+            'hardware/motor_on', Bool, queue_size=1, latch=True
+        )
+        self._e_stop_state_pub = rospy.Publisher(
+            'hardware/e_stop', Bool, queue_size=1, latch=True
+        )
```

Comment:
This can be one liners

Reply:
Black had different idea

Reply:
You sure you have line_length set to 100? My black makes it one line